repos/system_upgrade/el7toel8/libraries/fetch.py

```diff
@@ -21,6 +22,34 @@ def _raise_error(local_path, details):
     raise StopActorExecutionError(summary, details={'details': details, 'hint': hint})
 
 
+def _request_data(service_path, cert, proxies, timeout=REQUEST_TIMEOUT):
+    logger = api.current_logger()
+    attempt = 0
+    while True:
+        attempt += 1
+        try:
+            response = requests.get(service_path, cert=cert, proxies=proxies, timeout=REQUEST_TIMEOUT)
+            break
```

Comment:
```suggestion
            return requests.get(service_path, cert=cert, proxies=proxies, timeout=REQUEST_TIMEOUT)
```

Reply:
I was thinking about that, but I was afraid that linter will be blaming missing return in the end of the function. I will try it

Reply:
Well it would cause to be return None automatically (that's how it works in python)
However before that, you would end up raising an exception.

Reply:
There are only two exit points in this exception, that is the return and the re-raising of the exception

Reply:
I know, but linters have (had?) problems with it because of FP. Anyway, it seems it's passing like that.